tests/fixtures/files/valid_adms.rdf

```diff
@@ -47,6 +47,8 @@
         <dcat:landingPage rdf:resource="http://example.com"/>
         <dc:hasVersion rdf:resource="http://example.com/release/1"/>
         <dc:hasVersion rdf:resource="http://example.com/release/2"/>
+        <dct:relation rdf:resource="http://example.com/solution/3"/>
```

Comment:
Please, provide more than invalid entries for each field that can have invalid entries.

Reply:
@idimopoulos for each taxonomy term reference field?